pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -95,9 +95,17 @@ func syncOpenShiftControllerManager_v311_00_to_latest(c OpenShiftControllerManag
 
 	// our configmaps and secrets are in order, now it is time to create the DS
 	// TODO check basic preconditions here
+	var progressingMessages []string
 	actualDaemonSet, _, err := manageOpenShiftControllerManagerDeployment_v311_00_to_latest(c.kubeClient.AppsV1(), c.recorder, operatorConfig, c.targetImagePullSpec, operatorConfig.Status.Generations, forceRollout, c.proxyLister)
 	if err != nil {
-		errors = append(errors, fmt.Errorf("%q: %v", "deployment", err))
+		msg := fmt.Sprintf("%q: %v", "deployment", err)
```

Comment:
what's the goal here?

Reply:
to include error information in the progressing message